README.md

```diff
@@ -28,9 +28,21 @@ require('skywalking-nodejs').start({
 });
 ```
 
+### Download and deploy skywalking backend.
+It's very important that nodejs agent and Skywalking backbend that your download should be compatible. Please see [the document](docs/capatibility-list.md) and Here is the [deploy skywalking backend documents](https://github.com/apache/incubator-skywalking/blob/master/docs/en/Deploy-backend-in-standalone-mode.md)
```

Comment:
This is still guiding people to standalone mode deployment doc, which doesn't match the title. Also make the people confused, whether following this or document. Please consider to delete this.